bookwyrm/forms/landing.py

```diff
@@ -42,7 +42,10 @@ class Meta:
         model = models.User
         fields = ["localname", "email", "password"]
         help_texts = {f: None for f in fields}
-        widgets = {"password": forms.PasswordInput()}
+        widgets = {
+            "password": forms.PasswordInput(),
+            "preferred_timezone": forms.TextInput(),
```

Comment:
I don't think this is necessary since it's being used as a hidden field?

Reply:
I'll try tomorrow 👍 iirc Django complained without this being there

Reply:
thank you!

Reply:
I found some time tonight, and you were right, this isn't needed at all 🕺
I'm new to Django, so this might happen often until I get used to it after a while